### PR TITLE
PlanifNeige component documentation

### PR DESCRIPTION
**Pull request by kkr16**
**Description:** Documentation for new component. PlanifNeige to get street-by-street snow removal schedule and status from the city of Montreal's APIs.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#21421

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.planifneige.markdown [hunk 1]
```diff
@@ -20,33 +20,37 @@ In order to use the API, you will need to request an API from the [City of Montr
...
 
+Once you've obtained an API key, and have identified the streets you want to monitor, you can generate a `configuration.yaml` entry as shown in the below example:
 
-Then add the data to your `configuration.yaml` file as shown in the example:
-
-```yaml
+```
 # Example configuration.yaml entry
-sensor:
-  - platform: planifneige
```
Comment (klaasnicolaas) [inline source/_components/sensor.planifneige.markdown]: Why not `- platform: planifneige`?

Comment (kkr16) [reply source/_components/sensor.planifneige.markdown]: I had to re-write my code to separate it into a platform component, this is how I figured the config would be best..

Is there a preference for:

```
sensor:
  - platform: planifneige
    REST_OF_CONFIG
```

over:
```
planifneige:
  REST_OF_CONFIG
```

Comment (klaasnicolaas) [reply source/_components/sensor.planifneige.markdown]: I do not really know what the standard is at the moment, I asked more because it struck me.

Comment (kkr16) [reply source/_components/sensor.planifneige.markdown]: I based myself off of this https://github.com/home-assistant/architecture/issues/131

Comment (klaasnicolaas) [reply source/_components/sensor.planifneige.markdown]: Where is btw the Parent PR in Home Assistant repo?

Comment (kkr16) [reply source/_components/sensor.planifneige.markdown]: I'm facing some with the tox tests before opening the PR. 
Didn't have enough time to look into it due to work. Should have it done by this weekend.

Comment (kkr16) [reply source/_components/sensor.planifneige.markdown]: **Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** [home-assistant/home-assistant#21421](https://github.com/home-assistant/home-assistant/pull/21421)

I have also updated the PR accordingly.

_Edit: updated the parent PR#_

**Pull request closed without merging**